src/gui/sharee.cpp

```diff
@@ -100,6 +100,11 @@ void ShareeModel::shareesFetched(const QJsonDocument &reply)
             newSharees.append(parseSharee(group.toObject()));
         }
 
+        auto emails = exact.value("emails").toArray();
```

Comment:
is there a reason for emails not being const?

Reply:
@camilasan There isn't. Just did this in the same style as others. Then, I have to change others to "const" as well for not to introduce inconsistency?

Reply:
@kevin would say always use const :) I would do a separate commit to fix the others in this file or leave it to change when we actually have to touch those lines.

Reply:
@camilasan  Ok. Well, I'd say we change all to 'const' in this PR. No need to separately fix them. But, let's try to agree on what we do, on Monday maybe.

So, basically, I am fine if we change it everywhere, not just in the new lines.

Reply:
Sure.

Reply:
Agree with @camilasan since we got the nose on it and it's not too much work, let's have them all const and do that in a second commit in this same PR.